.github/workflows/api_test.yml

```diff
@@ -4,11 +4,14 @@
 name: Test schematic API
 
 on:
+  push:
+    branches: ['main', 'develop']
```

Comment:
Yes. I have made that change. But it is a bit difficult to test on my own since my test repo does not have a collaborator. Can I add you as a collaborator so that i could test out the workflow before merging?

Reply:
hmm maybe this is not needed. I could see in github action that the tests are running.

Reply:
I can see them running from my end as well

Reply:
@linglp Just a warning that the benchmarking tests will likely fail as everything stands right now. 
@milen-sage we should decide what our acceptable time is for the validation rules as things currently stand and set the tests to ensure execution time is below that, and then when we make the optimizations decrease the time to what we're targeting, otherwise the api tests will always show as failing and prevent merging without an override.

Reply:
I noticed that if I re-requested review. The tests don't get run again.